.gitpod.yml

```diff
@@ -22,6 +22,7 @@ tasks:
       export PIP_USER=no
       cd /workspace
       pip install -e /workspace/kedro --no-deps
+      source ~/.bashrc
```

Comment:
Would be nice to add a comment here explaining why it's needed (even if it's not really clear why...)